docs/features/feature-guides/properties/create-a-property.md

```diff
@@ -0,0 +1,221 @@
+   - **Property Type:** Select **Text**, allowing any valid string to be entered.  
+   - **Applies To:** Set this property to apply exclusively to **Columns**.  
+
+2. **Display Preferences:**  
+   - By default, column-level Structured Properties will be enabled for **all columns** on **all datasets** within DataHub, accessible via the Column Sidebar.  
+   - Optionally, enable **Show in Table Columns** to make the **Business Label** visible within the Columns Table on the dataset schema.  
+
+<p align="center">
+  <img 
+       width="90%"  
+       src="https://raw.githubusercontent.com/datahub-project/static-assets/refs/heads/main/imgs/structured_properties/5.2-create-structured-property-column-display-options.png"
+       alt="Configuring Column Structured Property"/>
+</p>
+
+:::caution
+While the column sidebar provides convenient access to assigned properties, adding too many Structured Properties can clutter the view. Limit the number of properties shown in the sidebar to maintain clarity and usability.
+:::
+
+Once configured, the **Business Label** Structured Property will automatically be added to all columns on dataset assets within DataHub.
+
+For example, after assigning the property, it will appear in two key areas of the `pet_profiles` Asset Page:
+
+1. **Columns Table:** The **Business Label** property and its populated values will be displayed directly within the Columns Table on the Dataset Schema, enabling users to view field-level metadata easily.  
+2. **Column Sidebar:** Structured Properties configured for columns, including **Business Label**, will also appear in the column’s sidebar.  
+
+By applying column-level Structured Properties like **Business Label**, you enhance data discoverability and provide business users with valuable insights while keeping the interface user-friendly.
+
+<p align="center">
+  <img 
+       width="90%"
+       src="https://raw.githubusercontent.com/datahub-project/static-assets/refs/heads/main/imgs/structured_properties/6.1-view-structured-property-column-level.png"
+       alt="Column-level Structured Property in Columns Table"/>
+</p>
+
+### Update the Business Label from the Column Sidebar  
+
+When selecting a specific column in the UI, the **Business Label** Structured Property will be visible in the column’s sidebar. Users with appropriate permissions can view or update the value directly from this interface.
+
+<p align="center">
+  <img 
+       width="90%"
+       src="https://raw.githubusercontent.com/datahub-project/static-assets/refs/heads/main/imgs/structured_properties/6.2-view-structured-property-column-level-sidebar.png"
+       alt="Column-level Structured Property in Sidebar"/>
+</p>
+
+This setup ensures that column-specific metadata, such as the **Business Label**, is accessible and actionable, helping business users better understand the dataset's structure and its alignment with key business concepts.
```

Comment:
Maybe small, less fleshed out sections for how to a) edit and b) delete a property once it's been created? What do you think?